WcaOnRails/app/views/layouts/application.html.erb

```diff
@@ -54,7 +54,18 @@
               <li><a href="<%= logo_path %>"><%= image_tag('wca_logo.svg', width: '16px', class: 'fa-fw') %> Logo</a></li>
             </ul>
           </li>
-          <li><a href="/results/competitions.php" class="top-nav"><%= icon('globe', class: 'fa-fw') %> Competitions</a></li>
+          <li class="dropdown">
+            <a href="" class="dropdown-toggle top-nav">
+               <%= icon('cube', class: 'fa-fw') %> Competitions <span class="caret"></span>
+            </a>
+            <ul class="dropdown-menu" role="menu">
+              <li><a href="/results/competitions.php"><%= icon('globe', class: 'fa-fw') %> All</a></li>
+              <li><a href="<%= my_comps_path %>"><%= icon('feed', class: 'fa-fw') %> My Competitions</a></li>
+              <% if current_user && current_user.any_kind_of_delegate? %>
```

Comment:
Actually, could we change this to `current_user.can_create_competition?` and add a test that people who are logged in cannot create a competition unless they're a delegate? See https://github.com/cubing/worldcubeassociation.org/issues/309